### DISCUSSION TITLE
Error with: $dispatch

### DISCUSSION THREAD

**Discussion by abr4xas** [wire-elements/spotlight#94] [Q&A] [answered]
I'm following the docs and this:

```html
<button @click="$dispatch('toggle-spotlight')">Toggle Spotlight</button>
```

it doesn't seem to work. 

I have a clean installation of laravel and spotlight.

**Comment by ManissDev** [comment, marked as answer]
You forgot to add x-data without it, it won't work

**Reply by abr4xas** [reply]
@ManissDev I will try that. Thanks.
